lib/reporters/postman/helpers/constants.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * An exhaustive set of constants used across various functions
+ */
+module.exports = {
+    /**
+     * Used as a source in the collection run object
+     */
+    NEWMAN_STRING: 'newman',
+
+    /**
+     * The status of the newman run in process
+     */
+    NEWMAN_RUN_STATUS_FINISHED: 'finished',
+
+    /**
+     * The success result of a particular test
+     */
+    NEWMAN_TEST_STATUS_PASS: 'pass',
+
+    /**
+     * The failure result of a particular test
+     */
+    NEWMAN_TEST_STATUS_FAIL: 'fail',
+
+    /**
+     * The skipped status of a particular test
+     */
+    NEWMAN_TEST_STATUS_SKIPPED: 'skipped',
+
+    /**
+     * Use this as a fallback collection name when creating collection run object
+     */
+    FALLBACK_COLLECTION_RUN_NAME: 'Collection Run',
+
+    /**
+     * The base URL for postman API
+     */
+    POSTMAN_API_BASE_URL: 'https://api.getpostman.com',
```

Comment:
Why are we using the older `getpostman.com` domain name?

Reply:
Switched to postman.com, thanks
Have also modified related regexs to support both `getpostman.com` and `postman.com`.